x-pack/plugins/observability_solution/observability_ai_assistant_management/public/routes/components/knowledge_base_edit_manual_entry_flyout.tsx

```diff
@@ -157,6 +169,34 @@ export function KnowledgeBaseEditManualEntryFlyout({
             onChange={(text) => setNewEntryText(text)}
           />
         </EuiFormRow>
+        <EuiSpacer size="m" />
+
+        <EuiFormRow fullWidth>
+          <EuiCheckbox
+            id="isPublicCheckbox"
+            checked={isPublic}
+            label={i18n.translate(
+              'xpack.observabilityAiAssistantManagement.knowledgeBaseEditManualEntryFlyout.euiCheckbox.isPublicLabel',
+              { defaultMessage: 'Public: Entry should be available to all users' }
+            )}
+            onChange={(e) => {
+              setIsPublic(e.target.checked);
+            }}
+          />
+        </EuiFormRow>
+        <EuiFormRow fullWidth>
+          <EuiCheckbox
+            id="isSystemPromptCheckbox"
+            checked={isSystemPrompt}
+            label={i18n.translate(
+              'xpack.observabilityAiAssistantManagement.knowledgeBaseEditManualEntryFlyout.euiCheckbox.isSystemPromptLabel',
+              { defaultMessage: 'System prompt: Entry should be included in the system prompt' }
```

Comment:
Same as above, change to "User instruction", or simply Instruction?

Reply:
Removed this per our conversation today.